deps/oob-1.0.2/jsonutil.h

```diff
@@ -36,6 +36,11 @@ struct logDetails{
   int urgent;
 };
 
+typedef struct dsnKeyValue{
```

Comment:
typedef struct KeyValuePair

a. Please capitalize type names
b. The type name is not specific to DSn, so just call is KeyBaluePair, or better yet
KeyValueStringPair